File: YaccConstructor/YC.ReSharper.AbstractAnalysis/YC.ReSharper.AbstractAnalysis.Plugin/AboutAction.cs
using System.Windows.Forms;
using JetBrains.ActionManagement;
using JetBrains.Application.DataContext;

namespace YC.ReSharper.AbstractAnalysis.Plugin.Core
{
  [ActionHandler("YC.ReSharper.AbstractAnalysis.Plugin.Core.About")]
  public class AboutAction : IActionHandler
  {
    public bool Update(IDataContext context, ActionPresentation presentation, DelegateUpdate nextUpdate)
    {
      // return true or false to enable/disable this action
      return true;
    }

    public void Execute(IDataContext context, DelegateExecute nextExecute)
    {
      MessageBox.Show(
        "ReSharper.AbstractAnalysis\nJetBrains Lab\n\nAbstract analysis for string-embedded languages.",
        "About ReSharper.AbstractAnalysis",
        MessageBoxButtons.OK,
        MessageBoxIcon.Information);
    }
  }
}

File: src/TSQL/AboutAction.fs
module AboutAction

//open JetBrains.ActionManagement
//open JetBrains.Application
//open JetBrains.UI.ActionsRevised
//open JetBrains.UI.Application
//open JetBrains.UI.ToolWindowManagement
//open System.Windows.Forms

//[<assembly: ActionsXml("YC.ReSharper.AbstractAnalysis.Languages.TSQL.Actions.xml")>]
//do ()

//[<ActionHandler("Plugins.TSQL.About")>]

//[<ToolWindowDescriptor(Text = "TSQL", 
//    VisibilityPersistenceScope = ToolWindowVisibilityPersistenceScope.Solution,
//    Type = ToolWindowType.SingleInstance,
//    InitialDocking = ToolWindowInitialDocking.NotSpecified)>]
//type WindowDescriptor (applicationHost : IApplicationHost) =
//    inherit ToolWindowDescriptor(applicationHost)
//
//type AboutAction () =
//    inherit ActivateToolWindowActionHandler<WindowDescriptor>()
//        member this.Update (context, presentation, nextUpdate) = true
//        member this.Execute (context, nextExecute) =
//            MessageBox.Show("TSQL plugin") |> ignore